docs/enterprise-appstore/enterprise-app-store-profile.md

```diff
@@ -0,0 +1,272 @@
+
+### Upload via API & CLI
+
+The above tasks can also be initiated by our Appcircle CLI. Please check the Appcircle CLI documentation for the command line parameters.
+
+<ContentRef url="/appcircle-api">Appcircle CLI</ContentRef>
+
+## Profile Actions
+
+Within the Enterprise App Store, you have several key actions that allow you to manage and interact with your profiles efficiently. Below are the descriptions of the available profile actions:
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4225-profile3.png' />
+
+#### 1. **Open**
+The "Open" action allows you to enter a specific profile, giving you access to all the details and settings associated with that profile. By selecting "Open," you can view and manage the apps, binaries, and configurations linked to the profile.
+
+#### 2. **Pin**
+The "Pin" action enables you to prioritize a profile by pinning it to the top of your profile list. When a profile is pinned, it remains easily accessible, especially if you manage multiple profiles. This action is particularly useful for frequently accessed profiles, ensuring they stay at the forefront of your workflow.
+
+#### 3. **Delete**
+The "Delete" action permanently removes a profile from the Enterprise App Store. Once a profile is deleted, all associated data, including uploaded binaries and settings, are also erased.
+
+:::info
+
+Unlike the profiles from other modules within the Appcircle, Enterprise App Store profiles cannot be named or renamed manually. The profiles will display the binary names instead.
+
+:::
+
+## Profile Settings
+
+After you have created your profile, it is time to configure it and send it to different users and channels.
+
+You can reach your profile information and add users for them to gain access to the Live and Beta channels by clicking the **Settings** button.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4225-profile4.png' />
+
+### Information
+
+You can copy your Profile ID from the Info tab by clicking the copy icon on the right side of your displayed ID.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-profilesettings.png" />
+
+### Manage Access
+
+You can add multiple users to the Beta and Live channels by hitting the Tab key.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-notify1.png" />
+
+:::important
+
+To configure the Manage Access tab, the authentication method for your Enterprise Store must be either SSO or LDAP. For more information about authentication types, please refer to the [Store Settings](/enterprise-appstore/store-settings#store-authentication) documentation.
+
+:::
+
+
+:::caution
+If you work for a sub-organization, you will only have visibility of the apps within that organization. However, if an app is published, it will also be displayed in the parent organization's showcase on the Enterprise App Store.
```

Comment:
```suggestion
If a sub-organization is being worked for, visibility will be limited to the apps within that organization. However, if an app is published, it will also be displayed in the showcase of the parent organization on the Enterprise App Store.
```